### PR TITLE
Use consistent changelog format

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,103 +1,114 @@
...
+
+<!-- New PRs should document their changes here. -->
+
+### [1.14.11] - 2016-04-11
 - Radically simpler inlining design. Move all imports and scripts into document
   body, replace imports inline.
 - Add tests for more complicated inlining scenarios
 
-### 1.14.10
+### [1.14.10] - 2016-04-07
```
Comment (justinfagnani) [inline CHANGELOG.md]: why brackets?

Comment (rictic) [inline CHANGELOG.md]: It's the keep a changelog convention. I don't have a strong opinion, I just want a convention.

Comment (justinfagnani) [inline CHANGELOG.md]: Seems like noise to me. If it doesn't serve a function, I'd drop it.

**Comment by rictic** [comment]
Thanks for the review!

**Pull request merged**